js/js.libraries/src/core/generated/_ArraysJs.kt

```diff
@@ -1767,18 +1812,63 @@ public inline fun CharArray.last(predicate: (Char) -> Boolean): Char {
     throw NoSuchElementException("Array contains no element matching the predicate.")
 }
 
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun <@kotlin.internal.OnlyInputTypes T> Array<out T>.lastIndexOf(element: T): Int {
+    return lastIndexOf(element, 0)
+}
+
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun ByteArray.lastIndexOf(element: Byte): Int {
+    return lastIndexOf(element, 0)
+}
+
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun ShortArray.lastIndexOf(element: Short): Int {
+    return lastIndexOf(element, 0)
+}
+
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun IntArray.lastIndexOf(element: Int): Int {
+    return lastIndexOf(element, 0)
+}
+
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun LongArray.lastIndexOf(element: Long): Int {
+    return lastIndexOf(element, 0)
+}
+
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun FloatArray.lastIndexOf(element: Float): Int {
+    return lastIndexOf(element, 0)
+}
+
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun DoubleArray.lastIndexOf(element: Double): Int {
+    return lastIndexOf(element, 0)
+}
+
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun BooleanArray.lastIndexOf(element: Boolean): Int {
+    return lastIndexOf(element, 0)
+}
+
+@Deprecated("Use lastIndexOf(T, Int)", level = DeprecationLevel.HIDDEN)
+public fun CharArray.lastIndexOf(element: Char): Int {
+    return lastIndexOf(element, 0)
+}
+
 /**
  * Returns last index of [element], or -1 if the array does not contain element.
  */
-public fun <@kotlin.internal.OnlyInputTypes T> Array<out T>.lastIndexOf(element: T): Int {
+public fun <@kotlin.internal.OnlyInputTypes T> Array<out T>.lastIndexOf(element: T, startIndex: Int = 0): Int {
```

Comment:
In string extensions `lastIndexOf` begins search from `startIndex` position (which is `lastIndex` by default) and goes down to zero.